core/src/main/java/org/mskcc/cbio/oncokb/model/IndicatorQueryOncogenicity.java

```diff
@@ -0,0 +1,19 @@
+package org.mskcc.cbio.oncokb.model;
+
+public class IndicatorQueryOncogenicity {
+    Oncogenicity oncogenicity;
```

Comment:
Since this got moved out of `IndicatorUtils.java` and into a different package, would it make sense to use the `private` access modifier instead of the default?

Reply:
Good point, let me fix